site/content/docs/main/restore-reference.md

```diff
@@ -215,37 +215,9 @@ data:
 
 ### PVC selected-node
 
-Velero by default removes PVC's `volume.kubernetes.io/selected-node` annotation during restore, so that the restored PVC could be provisioned appropriately according to ```WaitForFirstConsumer``` rules, storage topologies and the restored pod's schedule result, etc.  
```

Comment:
Reading this line makes me feel that maybe this feature has some value.
Ideally, if the ConfigMap doesn't exist or no new node is found, the annotation should be deleted.
In contrast, keeping it.